### PR TITLE
add nav(sidebar) with tailwindcss

### PR DESCRIPTION
**Pull request by cazterk**
- [x] add sidebar

### CODE REVIEW

**Comment by cazterk** [comment]
- [x] add sidebar navigation
Peek 2021-11-15 07-44

File: src/pages/location.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <div className=" h-screen">
+      {' '}
+      <div className="flex  flex-col  w-full h-full ">
+        <div className={styles.grad}>
+          <div className="">
+            <h1>Location</h1>
+            <div className="flex ">
+              <Image
+                className=" justify-center z-10 "
+                src="/map.svg"
```
Comment (sonlinux) [inline src/pages/location.tsx:18]: we can not have a map as an svg @cazterk 
you might want to look through some examples 
https://cheatcode.co/tutorials/how-to-render-a-map-with-markers-using-google-maps-in-next-js

File: src/pages/location.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+                width={900}
+                height={600}
+              />
+            </div>
+            <ul>
+              <li>Lat/Long: -15, 30 </li>
+              <li>Province: Lusaka</li>
+              <li>District: Chongwe</li>
+              <li>Service Area: Chongwe</li>
```
Comment (sonlinux) [inline src/pages/location.tsx:27]: we need these as map pop ups and not using  the default pointer

File: src/styles/location.module.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.grad {
+  /* background: linear-gradient(
+    180deg,
+    #fbc7c7 0%,
+    #a9c9d4 17.66%,
+    #ede0e6 63.23%,
+    rgba(236, 224, 229, 0.28) 84.91%,
+    rgba(169, 201, 212, 0) 100%
+  ); */
```
Comment (sonlinux) [inline src/styles/location.module.css:9]: if we arent using the code then we do  not need keeping it

**Review by sonlinux — APPROVED** [review]
for the routing  am not pretty comfortable hardcoding the routes that way
it would be nice if we can make them dynamic rather than having to drill down into code to  just update a route

**Review by sonlinux — CHANGES_REQUESTED** [review]
Ensure you sync with @wearethepioneers18 on this layout

File: src/pages/location.tsx (hunk 1 above)
Comment (cazterk) [reply src/pages/location.tsx:18]: that was just a placeholder to test out the layout and the ticket for the location is on prince's side

**Comment by cazterk** [comment]
- [x] close pr

**Pull request closed without merging**